src/zyppinfo.h

```diff
@@ -0,0 +1,68 @@
+#ifndef ZYPPINFO_H
+#define ZYPPINFO_H
+
+
+#include <zypp/ResKind.h>
+#include <zypp/base/Algorithm.h>
+#include <zypp/ZYppFactory.h>
+#include <zypp/ResPool.h>
+#include <zypp/PoolQuery.h>
+#include <zypp/base/Easy.h>
+#include "utils.h"
+
+class Info 
+{
+private:
+    string m_repositoryName;
+    string m_packageName;
+    string m_version;
+    string m_arch;
+    string m_vendor;
+    string m_installed;
+    string m_status;
+    string m_installedSize;
+public:
+    string repository() { return m_repositoryName; }
+    string packageName() { return m_packageName; }
+    string version() { return m_version; }
+    string arch() { return m_arch; }
+    string vendor() { return m_vendor; }
+    string installed() { return m_installed; }
```

Comment:
what do you think about renaming this to isInstalled() and returning a bool ? That way it's easier to use in other parts of the project and doesn't depend on the language